src/connectedvmware/azext_connectedvmware/_help.py

```diff
@@ -638,6 +637,63 @@
                --vm-name "name of the vm"
 """
 
+helps['connectedvmware vm extension'] = """
+    type: group
+    short-summary: Manage vm extension with connectedvmware
+"""
+
+helps['connectedvmware vm extension list'] = """
+    type: command
+    short-summary: "The operation to get all extensions of a non-Azure vm."
+    examples:
+      - name: Get all VM Extensions
+        text: |-
+               az connectedvmware vm extension list --vm-name "vm name" --resource-group "myResourceGroup"
+"""
+
+helps['connectedvmware vm extension show'] = """
+    type: command
+    short-summary: "The operation to get the extension."
+    examples:
+      - name: Get VM Extension
+        text: |-
+               az connectedvmware vm extension show --name "CustomScriptExtension" --vm-name "vm name" \
+--resource-group "myResourceGroup"
+"""
+
+helps['connectedvmware vm extension create'] = """
+    type: command
+    short-summary: "The operation to Create the extension."
+    examples:
+      - name: Create or Update a VM Extension
```

Comment:
I'm a bit confused about `Create or Update`, I think create and update are independent ?

Reply:
Both are different, internally we are using different API. I have updated the help des.